Sources/MongoSwift/MongoClient.swift

```diff
@@ -6,9 +6,15 @@ public struct ClientOptions: CodingStrategyProvider, Decodable {
     /// Determines whether the client should retry supported write operations.
     public var retryWrites: Bool?
 
-    /// Indicates whether this client should be set up to enable monitoring command and server discovery and monitoring
-    /// events.
-    public var eventMonitoring: Bool
+    /// Indicates whether this client should publish command monitoring events.
+    public var commandMonitoring: Bool = false
+
+    /// Indicates whether this client should publish server monitoring events.
+    public var serverMonitoring: Bool = false
```

Comment:
I think it would be useful to document which events are covered by which flag, similar to how it's documented on the `MongoEventType` enum.

Reply:
done